submissions/korotushak96/js-oop/js/app.js

```diff
@@ -0,0 +1,94 @@
+// Enemies our player must avoid
+const enemyWidth = 101;
+const fieldWidth = 504;
+const fieldHeight = 385;
+const finishY = 0;
+const stepY = 85;
+const stepX = 101;
+const startX=202;
+const startY=385;
+
+var Enemy = function(x,y) {
+    this.speed = Math.floor(Math.random() * (350 - 150 + 1)) + 150;
```

Comment:
`350`, `150` etc. it's a magic numbers. You can define constants for that. Also check yourself in code below.